clients/admin-ui/src/features/common/nav/v2/BackButton.tsx

```diff
@@ -15,9 +15,7 @@ const BackButton = ({
   backPath,
   ...props
 }: { backPath: string } & FlexProps) => (
-  // In general Layout provides a default top padding. But on pages with
-  // Back buttons, we want the Back button to be higher and for the page to be less top-padded
-  <Flex alignItems="center" mt={-4} mb={3} {...props}>
+  <Flex alignItems="center" mb={6} {...props}>
```

Comment:
Opinionated change here. I know the intention was to make this less "top-padded" but I didn't think it looked good for the back button to be so close to the top of the page, this is what it looks like now.
<img width="960" alt="image" src="https://github.com/ethyca/fides/assets/2092272/8fb2d046-38d4-44df-9995-538e7eec595a">